LayoutTests/css3/unicode-bidi-float-crash.html

```diff
@@ -0,0 +1,20 @@
+<style>
+div {
+  float: left;
+}
+</style>
+<span style="unicode-bidi: isolate">
+  <span style="unicode-bidi: isolate">
+    <span id=inline_box>
+      <div></div>
+    </span>
+  </span>
+</span>
```

Comment:
I'd completely collapse this content

`</style><span style="unicode-bidi: isolate"><span style="unicode-bidi: isolate"><span id=inline_box><div></div></span></span></span>`

to remove RenderText noise from the output. it may also eliminate the need for platform specific dumps.